Gems/Atom/RPI/Code/Include/Atom/RPI.Public/DynamicDraw/DynamicDrawContext.h

```diff
@@ -229,10 +254,21 @@ namespace AZ
             // Draw variations allowed in this DynamicDrawContext
             DrawStateOptions m_drawStateOptions;
 
-            // For generate output attachment layout and filter draw items
-            Scene* m_scene = nullptr;
+            // DrawListTag used to help setup PipelineState's output
+            // and also for submitting draw items to views 
             RHI::DrawListTag m_drawListTag;
 
+            // Output scope related
+            enum class OutputScopeType
+            {
+                Scene = 0,
+                RenderPipeline,
+                RasterPass
+            };
```

Comment:
Tiny nit: I would add a "Unset" value just for initialization to help with debugging.